Services/DeviceClient.cs

```diff
@@ -30,6 +33,7 @@ public class DeviceClient : IDeviceClient
 
         // See also https://github.com/Azure/toketi-iothubreact/blob/master/src/main/scala/com/microsoft/azure/iot/iothubreact/MessageFromDevice.scala
         private const string CREATION_TIME_PROPERTY = "$$CreationTimeUtc";
+        private const string CLASSNAME_PROPERTY = "$$ClassName";
```

Comment:
this is not used, is it? perhaps the method `SendProtoMessageAsync` was supposed to add a message property with this name?

Reply:
It is used in line 238